src/main/java/com/ciandt/techgallery/service/impl/CronServiceImpl.java

```diff
@@ -161,5 +160,46 @@ private Date findLastExecutedCronJob() {
     return lastCronJobExecDate;
   }
 
+  @Override
+  public void sendEmailToEndorseds() {
+    CronJob cronJob = new CronJob();
```

Comment:
@tsr-thulio @brunoliberal What do you think about creating an annotation to cron methods similar to @transactional ?